go/vt/tableacl/tableacl_test.go

```diff
@@ -74,6 +74,21 @@ func TestInitWithValidConfig(t *testing.T) {
 	}
 }
 
+func TestInitWithEmptyConfig(t *testing.T) {
+	tacl := tableACL{factory: &simpleacl.Factory{}}
+	f, err := os.CreateTemp("", "tableacl")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(f.Name())
+	if err := f.Close(); err != nil {
+		t.Fatal(err)
```

Comment:
We should use require/assert in new tests. So e.g. here it would be:
```
err := f.Close()
require.NoError(t, err)
```

Or even just: `require.NoError(t, f.Close())`